diff -up DevIL/DevIL/src-IL/src/il_manip.cpp.omv~ DevIL/DevIL/src-IL/src/il_manip.cpp
--- DevIL/DevIL/src-IL/src/il_manip.cpp.omv~	2023-04-29 03:43:10.787303045 +0200
+++ DevIL/DevIL/src-IL/src/il_manip.cpp	2023-04-29 03:43:17.402357747 +0200
@@ -37,9 +37,9 @@ ILushort ILAPIENTRY ilFloatToHalf(ILuint
 	// of float and half (127 versus 15).
 	//
 
-	register int s =  (i >> 16) & 0x00008000;
-	register int e = ((i >> 23) & 0x000000ff) - (127 - 15);
-	register int m =   i        & 0x007fffff;
+	int s =  (i >> 16) & 0x00008000;
+	int e = ((i >> 23) & 0x000000ff) - (127 - 15);
+	int m =   i        & 0x007fffff;
 
 	//
 	// Now reassemble s, e and m into a half:
